<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mpx3GUI</class>
 <widget class="QMainWindow" name="Mpx3GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mpx3GUI</string>
  </property>
  <property name="toolTip">
   <string>Visualization</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QCstmGLVisualization" name="visualizationGL"/>
      <widget class="QCstmConfigMonitoring" name="CnMWidget"/>
      <widget class="QCstmDacs" name="DACsWidget"/>
      <widget class="QCstmEqualization" name="equalizationWidget"/>
      <widget class="QCstmThreshold" name="ThresholdTab"/>
      <widget class="QCstmCT" name="ctTab"/>
      <widget class="QCstmVoxeltab" name="voxelTab"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMpx3Eq">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_data"/>
    <addaction name="actionSave_data"/>
    <addaction name="actionClear_data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QMenu" name="menuLoad_configuration">
     <property name="title">
      <string>Load configuration</string>
     </property>
     <addaction name="actionLoad_DACs"/>
     <addaction name="actionLoad_Equalization"/>
    </widget>
    <widget class="QMenu" name="menuSave_configuration">
     <property name="title">
      <string>Save configuration</string>
     </property>
     <addaction name="actionSave_DACs"/>
     <addaction name="actionSave_Equalization"/>
    </widget>
    <addaction name="actionClear_configuration"/>
    <addaction name="menuLoad_configuration"/>
    <addaction name="menuSave_configuration"/>
    <addaction name="separator"/>
    <addaction name="actionWizard"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionDefibrilator"/>
   </widget>
   <addaction name="menuMpx3Eq"/>
   <addaction name="menuSystem"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionSave_data"/>
   <addaction name="actionOpen_data"/>
   <addaction name="actionClear_data"/>
   <addaction name="separator"/>
   <addaction name="actionWizard"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear_configuration">
   <property name="text">
    <string>Clear configuration</string>
   </property>
  </action>
  <action name="actionLoad_DACs">
   <property name="text">
    <string>Load Config+DACs</string>
   </property>
  </action>
  <action name="actionLoad_Equalization">
   <property name="text">
    <string>Load Equalization</string>
   </property>
  </action>
  <action name="actionSave_DACs">
   <property name="text">
    <string>Save Config+DACs</string>
   </property>
  </action>
  <action name="actionSave_Equalization">
   <property name="text">
    <string>Save Equalization</string>
   </property>
  </action>
  <action name="actionSave_data">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Actions-document-save-icon.png</normaloff>:/icons/Actions-document-save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save data</string>
   </property>
  </action>
  <action name="actionOpen_data">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Actions-document-open-icon.png</normaloff>:/icons/Actions-document-open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open data</string>
   </property>
  </action>
  <action name="actionSumming">
   <property name="text">
    <string>summing</string>
   </property>
  </action>
  <action name="actionDiscrete">
   <property name="text">
    <string>discrete</string>
   </property>
  </action>
  <action name="actionClear_data">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Actions-edit-clear-icon.png</normaloff>:/icons/Actions-edit-clear-icon.png</iconset>
   </property>
   <property name="text">
    <string>Clear data</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Button-Turn-On-icon.png</normaloff>
     <normalon>:/icons/switch-turn-off-icon.png</normalon>:/icons/Button-Turn-On-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/switch-turn-off-icon.png</normaloff>:/icons/switch-turn-off-icon.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDefibrilator">
   <property name="text">
    <string>Defibrillator</string>
   </property>
  </action>
  <action name="actionWizard">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Actions-tools-wizard-icon.png</normaloff>:/icons/Actions-tools-wizard-icon.png</iconset>
   </property>
   <property name="text">
    <string>Wizard</string>
   </property>
   <property name="toolTip">
    <string>Configure the system with a wizard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCstmEqualization</class>
   <extends>QWidget</extends>
   <header>qcstmequalization.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCstmDacs</class>
   <extends>QWidget</extends>
   <header>qcstmdacs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCstmGLVisualization</class>
   <extends>QWidget</extends>
   <header>qcstmglvisualization.h</header>
   <container>1</container>
   <slots>
    <slot>on_frame_added()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QCstmThreshold</class>
   <extends>QWidget</extends>
   <header>qcstmthreshold.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCstmVoxeltab</class>
   <extends>QWidget</extends>
   <header>qcstmvoxeltab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCstmConfigMonitoring</class>
   <extends>QWidget</extends>
   <header>qcstmconfigmonitoring.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCstmCT</class>
   <extends>QWidget</extends>
   <header>qcstmct.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>frame_added()</signal>
  <signal>availible_gradients_changed(QStringList)</signal>
  <signal>gradient_added(QString)</signal>
  <signal>data_cleared()</signal>
 </slots>
</ui>
